File: src/Enemies/Enemies.Content/Content/Scripts/Entities/sample.lua
script = {}

function script.Initialize()
	entity:SetSpritesheet("Sprites/Human")
end

function script.DoUpdate(delta)
	entities = entity:GetNearbyEnemies()

	for value in array.foreach(entities) do
		-- Do Something
	end

	entity:Move(1,1)
end

return script

File: src/Enemies/Enemies.Content/Content/Scripts/Entities/sample.py
from Enemies.Entities import BaseEntity

class ScriptEntity(BaseEntity):
	def Initialize(self):
		self.SetSpritesheet("Sprites/Human")
		self.SetCurrentAnimation("stopped_down")
	
	def DoUpdate(self, game_time):
		super(ScriptEntity, self).DoUpdate(game_time)
